File: 2997/2997.cpp
class Solution {
public:
    int minOperations(vector<int>& nums, int k) {
        int sum=k;
        int result = 0;
        for(int num: nums){
            sum ^= num;
        }
        while(sum>0){
            if(sum&1) result++;
            sum = sum >>1;
        }
        return result;
    }
};

File: 2997/2997.js
/**
 * @param {number[]} nums
 * @param {number} k
 * @return {number}
 */
var minOperations = function (nums, k) {
  let sum = k;
  let result = 0;
  for (const num of nums) {
    sum ^= num;
  }
  const value = sum.toString(2);
  result = value.split('1').length - 1;
  return result;
};
